### PR TITLE
completed update functionality formatted pages to read easier

### PR DESCRIPTION
**Pull request by Trzy426**

### CODE REVIEW

File: app/javascript/components/pages/FateNew.js [hunk 1]
```diff
@@ -29,8 +29,18 @@ class FateNew extends Component {
   };
   handleSubmit = (e) => {
     e.preventDefault();
-    this.props.createNewEvent(this.state.form);
-    this.setState({ submitted: true });
+    const { description, url, category, is_public } = this.state.form;
+    if (
+      (!description && description === "") ||
```
Comment (archae0pteryx) [inline app/javascript/components/pages/FateNew.js:34]: These can be simplified to just `!description`, etc...

File: app/javascript/components/pages/FateIndex.js [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-            <>
-                <div>
-                    <h2>This is Fate Index Page Test1</h2>
-                    {this.props.events && this.props.events.map((event, i) => <CardComponent event={event} key={i} />)}
-                    <Button title="Add an interest" handleClick={this.handleClick} />
-                </div>
-            </>
-        );
-    }
+  handleClick = () => {
```
Comment (iameddieyayaya) [inline app/javascript/components/pages/FateIndex.js:8]: This can be removed.

File: app/javascript/components/pages/FateUpdate.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  handleChange = (e) => {
+    const { form } = this.state;
+    form[e.target.name] = e.target.value;
+    this.setState({ form: form });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { description, url, category, is_public } = this.state.form;
+    if (
+      (!description && description === "") ||
```
Comment (archae0pteryx) [inline app/javascript/components/pages/FateUpdate.js:30]: You can simplifiy these too

**Review by iameddieyayaya — APPROVED** [review]

File: app/javascript/components/pages/FateNew.js (hunk 1 above)
Comment (Trzy426) [reply app/javascript/components/pages/FateNew.js:34]: you actually cant it does not work unless both are used in conjunction, i have no idea why tried both of them separately and they only work together

File: app/javascript/components/pages/FateUpdate.js (hunk 3 above)
Comment (Trzy426) [reply app/javascript/components/pages/FateUpdate.js:30]: same as above

File: app/javascript/components/pages/FateIndex.js (hunk 2 above)
Comment (Trzy426) [reply app/javascript/components/pages/FateIndex.js:8]: done

**Review by archae0pteryx — APPROVED** [review]

**Pull request closed without merging**